referenceframe/model.go

```diff
@@ -189,6 +190,18 @@ func (m *SimpleModel) MarshalJSON() ([]byte, error) {
 	return json.Marshal(m.modelConfig)
 }
 
+func (m *SimpleModel) ModelPieceFrames(inputs []Input) (map[string]*staticFrame, error) {
```

Comment:
What is this for?

Reply:
This allows us to get the positions of the individual pieces of the model, e.g. a specific joint.

Necessary for load calculation.